source/c/hello-world/CMakeLists.txt

```diff
@@ -4,6 +4,6 @@
 #------------------------------------------------------------------------------#
 
 add_executable(adios2-hello-world-c hello-world.c)
-target_link_libraries(adios2-hello-world-c adios2::adios2 MPI::MPI_C)
+target_link_libraries(adios2-hello-world-c adios2::c_mpi)
```

Comment:
This should still add `MPI::MPI_C` since the "application" has a direct dependency on it, not just a transitive one through adios.

Reply:
@chuckatkins : Yup, fixed.